docs/docs/basic-usage.md

```diff
@@ -41,6 +41,15 @@ python = "*"
 pytest = "^3.4"
 ```
 
+If you have a pre-existing project, say `pre-existing-demo`, and want to use poetry:
+
+```bash
+cd pre-existing-demo
+poetry init
+```
+
+This will interactively guide you through creating a `pyproject.toml` file.
+
```

Comment:
Thanks a lot your contribution @8F3E :+1: 

It's a good idea to show the `poetry init` way in the "basic usage" as well. But I think it should be separate headline and should include some words about the folder structure that `poetry` expects.

fin swimmer

Reply:
I've added the new headline and a bit about folder structure, not quite sure if it's in the right place in the file though.